tasks/task757_msr_sqa_question_generation.json

```diff
@@ -0,0 +1,6347 @@
+{
+    "Contributors": [ "Shreya Sarnaik sjsarnai@asu.edu" ],
+    "Categories": [ "Question Generation" ],
+    "Source": [ "Microsoft Research Sequential Question Answering (Dataset - https://huggingface.co/datasets/msr_sqa#source-data) (Paper - https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/acl17-dynsp.pdf)" ],
+    "Definition": "In this task, you are given the data in tabular/csv format and you must generate a question based on the information present in the csv file. Construct a question in such a way that 1) It is unambiguous, 2) It is answerable from the passage, 3) Its answer is unique, 4) Its answer is a continuous text span from the passage. Avoid creating questions that 1) Can be answered correctly without reading/understanding the passage, 2) Uses same phrases/words form the given passage.",
+    "Positive Examples": [
+        {
+            "input": "Passage: ['1', 'Ben McDonald', 'Baltimore Orioles', 'RHP', 'Louisiana State University']. ['2', 'Tyler Houston', 'Atlanta Braves', 'C', 'Valley HS (Las Vegas, NV)']. ['3', 'Roger Salkeld', 'Seattle Mariners', 'RHP', 'Saugus (CA) HS']. ['4', 'Jeff Jackson', 'Philadelphia Phillies', 'OF', 'Simeon HS (Chicago, IL)']. ['5', 'Donald Harris', 'Texas Rangers', 'OF', 'Texas Tech University']. ['6', 'Paul Coleman', 'Saint Louis Cardinals', 'OF', 'Frankston (TX) HS']. ['7', 'Frank Thomas', 'Chicago White Sox', '1B', 'Auburn University']. ['8', 'Earl Cunningham', 'Chicago Cubs', 'OF', 'Lancaster (SC) HS']. ['9', 'Kyle Abbott', 'California Angels', 'LHP', 'Long Beach State University']. ['10', 'Charles Johnson', 'Montreal Expos', 'C', 'Westwood HS (Fort Pierce, FL)']. ['11', 'Calvin Murray', 'Cleveland Indians', '3B', 'W.T. White High School (Dallas, TX)']. ['12', 'Jeff Juden', 'Houston Astros', 'RHP', 'Salem (MA) HS']. ['13', 'Brent Mayne', 'Kansas City Royals', 'C', 'Cal State Fullerton']. ['14', 'Steve Hosey', 'San Francisco Giants', 'OF', 'Fresno State University']. ['15', 'Kiki Jones', 'Los Angeles Dodgers', 'RHP', 'Hillsborough HS (Tampa, FL)']. ['16', 'Greg Blosser', 'Boston Red Sox', 'OF', 'Sarasota (FL) HS']. ['17', 'Cal Eldred', 'Milwaukee Brewers', 'RHP', 'University of Iowa']. ['18', 'Willie Greene', 'Pittsburgh Pirates', 'SS', 'Jones County HS (Gray, GA)']. ['19', 'Eddie Zosky', 'Toronto Blue Jays', 'SS', 'Fresno State University']. ['20', 'Scott Bryant', 'Cincinnati Reds', 'OF', 'University of Texas']. ['21', 'Greg Gohr', 'Detroit Tigers', 'RHP', 'Santa Clara University']. ['22', 'Tom Goodwin', 'Los Angeles Dodgers', 'OF', 'Fresno State University']. ['23', 'Mo Vaughn', 'Boston Red Sox', '1B', 'Seton Hall University']. ['24', 'Alan Zinter', 'New York Mets', 'C', 'University of Arizona']. ['25', 'Chuck Knoblauch', 'Minnesota Twins', '2B', 'Texas A&M University']. ['26', 'Scott Burrell', 'Seattle Mariners', 'RHP', 'Hamden (CT) HS']",
```

Comment:
How about we change "Passage" to "Table"?

Reply:
Are there any headers for these tables? Sth like, `[#, name, location, ...]`

Reply:
Yes, there are headers. I can include it in input field. But modifying that will need some time.

Reply:
Yeah, I think we need those; otherwise, it's difficult for one to interpret the semantics of the values in the tables.